### ISSUE TITLE
"Link not found" occurs when trying to set VF Admin mac

### ISSUE THREAD

**Issue by martinkennelly** [openshift/sriov-network-operator#335]
Hi, 
NIC: X700
driver: i40e
version: 2.10.19.82
firmware-version: 6.80 0x80003c64 1.2007.0
Operator version: Latest
K8s: Vanilla

I have noticed “Link not found” error occurs regularly. When the error occurs, and the logic flow restarts, it works fine then.
I was going to dig deeper on this but first I am wondering if you have encountered it when you use the SyncNodeState() function specially the setVfsAdminMac() function?

setVfsAdminMac(): unable to get VF link for device &{Name:ens785f0 Mac:68:05:ca:2d:ea:f0 Driver:i40e PciAddress:0000:02:00.0 Vendor:8086 DeviceID:1572 Mtu:1500 NumVfs:0 LinkSpeed:-1 Mb/s LinkType:ETH TotalVfs:32 VFs:[]} "Link not found"

I think adding a retry function here may help prevent returning an error and restarting the Apply() flow. What you think?

**Comment by zshi-redhat** [comment]
@martinkennelly which version have you encountered this issue?
I think we fixed a similar issue in this PR: https://github.com/openshift/sriov-network-operator/pull/314 , it is available if you deploy with latest master.

**Comment by martinkennelly** [comment]
I was using a version just slightly older than this commit. Let me try again. Thank you!

**Comment by martinkennelly** [comment]
Consuming PR #314 fixed the issue! Closing.
